Failure on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.39929&attempt_id=94645d86-197f-4f71-80b7-0e3064833e4b)

Attempted: john

<details><summary>Partial log (click to expand)</summary><p>

```
 extern size_t strlen (const char *__s)
               ^~~~~~
truecrypt_fmt_plug.c:314:95: error: 'num_iterations' undeclared (first use in this function)
    pbkdf2_sha512((const unsigned char*)key_buffer[i], strlen(key_buffer[i]), psalt->salt, 64, num_iterations, key, sizeof(key), 0);
                                                                                               ^~~~~~~~~~~~~~
make[1]: *** [Makefile:520: truecrypt_fmt_plug.o] Error 1
make[1]: Leaving directory '/build/john-1.8.0-jumbo-1/src'
make: *** [Makefile:185: default] Error 2
builder for '/nix/store/h4w2nw75xh967jsq8pi7x3hhbmla5aml-john-1.8.0-jumbo-1.drv' failed with exit code 2
[31;1merror:[0m build of '/nix/store/h4w2nw75xh967jsq8pi7x3hhbmla5aml-john-1.8.0-jumbo-1.drv' failed
```
</p></details>

--- a/pkgs/tools/security/john/default.nix
+++ b/pkgs/tools/security/john/default.nix
@@ -69,6 +69,6 @@ stdenv.mkDerivation rec {
     license = licenses.gpl2;
     homepage = https://github.com/magnumripper/JohnTheRipper/;
     maintainers = with maintainers; [ offline ];
-    platforms = platforms.unix;
+    platforms = [ "x86_64-linux" "x86_64-darwin"];
   };
 }

